_docs/LitmusVagrantTesting.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Testing Puppet Modules with Litmus and Vagrant
+---
+
+## Testing Puppet Modules with Litmus and Vagrant
+ 
+### The Setup 
+
+For this demo I'm gonna run through the entire process from start to finish - I hope that you find it useful.
+To start, the first thing that you need to do is clone down the module that you want to test; in this case we'll be using `puppetlabs-scheduled_task`. This can easily be accomplished with a simple command:
+
+```
+git clone https://github.com/user-name/puppetlabs-scheduled_task.git
+```
+
+Now that you have the module cloned down, simply run the below command to set up the environment:
+
+```
+bundle install --path .bundle/gems/
+```
+
+### Provisioning the Machine
+
+With the environment is setup it is time to provision the machine. Rather than spin up the vagrant machine directly, we can use Litmus as an intermediary to do it for us.
```

Comment:
```suggestion
With the environment setup it is time to provision the machine. Rather than spin up the vagrant machine directly, we can use Litmus as an intermediary to do it for us.
```

Reply:
Done